libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/router/RouterInterfaceAdapter.kt

```diff
@@ -124,12 +123,39 @@ class RouterInterfaceAdapter(
 
     override fun getRouteRefresh(
         options: RouteRefreshOptions,
-        route: String,
+        callback: RouterRefreshCallback
```

Comment:
I don't see in NN release notes why they remove the `route` param, do you know @LukasPaczos ?